### ISSUE TITLE
[concept] Quick Editor Pop-up Panel

### ISSUE THREAD

**Issue by 1280px** [florisboard/florisboard#1613]
`TLDR: add a pop-up panel with quick edit actions (such as copy, paste, move cursor etc. etc.) on utility key press.`

_Let's change the utility key to be actually useful!_

![pic1](https://user-images.githubusercontent.com/71165491/156590114-f8b86cef-5aed-4f89-9c44-f75943e182b5.png)

...this UI suggestion may look strange at first, but please read it to the end and you'll likely change your opinion completely.

For now, all the text formatting buttons are scattered across 2 panels and if you want to have both of them, you have to either sacrifice your visible screen height or make additional taps to open/close the second row. If you're typing with autocorrect on, you also have to make additional taps just to use the "Undo" button!

The Quick Editor pop-up, though, features all the useful text formatting buttons in one small window, close to each other and always just one tap away!

![pic2](https://user-images.githubusercontent.com/71165491/156594945-ebfe0616-03db-4eb6-987d-71ec60b7b376.png)

The concept was inspired by NextApp Keyboard's FN button, but instead of a separate screen for keys (and a key just to open/close this screen), the pop-up appears on a long tap on the emoji/language selection key (which currently only provides keyboard change on hover). It also has only necessary keys without ones used by developers.

A little more detailed buttons actions explanation:

![pic3](https://user-images.githubusercontent.com/71165491/156594953-289a3cc2-cbbd-4f5d-b5bb-aa317d7f81ff.png)

Of course, implementing this feature doesn't mean other ways of using the keyboard should be scrapped (even though it was meant to be an ultimate additional row replacement), but providing this as an option will be very welcomed by people who constantly use formatting tools.

_____________________________________________
P.s. I also made a few other concepts before the one I showcased above, which might be interesting too (though IMO the final one features the best UX):

https://user-images.githubusercontent.com/71165491/156595383-4f2e8300-8ab4-4208-8c70-21187cb48bff.png
(an original idea with 2 row-high grid being used)

https://user-images.githubusercontent.com/71165491/156595389-551e1177-a0cd-487c-a4a4-d1bbdd110427.png
(pop-up is aligned to the left, a little too close to the hand)

https://user-images.githubusercontent.com/71165491/156595391-61ae40a3-9c89-4df1-b5bc-79586453b5ed.png
(different buttons location)

**Comment by freMea** [comment]
I really miss the editor panel panel from gboard. Your concept would replace it if the pop-up is sticky. But I sincerely don’t care if the keys are available from a pop-up or another panel as long as it can be called with only one key press.
![image](https://user-images.githubusercontent.com/22337329/193487828-c704f2a8-6741-4460-8552-968955edef3a.png)

**Comment by 1280px** [comment]
Yeah, I've been using Technical Keyboard for a long time and I really miss its FN panel, which originally inspired me for this concept.
Revisiting this concept now, I think the pop-up should be a little wider, maybe up to 100% of the keyboard's width.
